PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.java

```diff
@@ -215,7 +262,7 @@ protected boolean isBasicAuthSelected() {
         return ((String) authComboBox.getSelectedItem()).equalsIgnoreCase("Basic Authentication");
     }
 
-    protected void basicValidate() {
+    protected void basicValidation() {
```

Comment:
Let the method name be `doSomething` patten:
```suggestion
    protected void validateBasicInputs() {
```